src/lib.zig

```diff
@@ -3026,6 +3034,49 @@ pub const Lua = struct {
         _ = c.luaopen_bit32(lua.state);
     }
 
+    /// Returns if given typeinfo is a string type
+    fn isTypeString(typeinfo: std.builtin.Type.Pointer) bool {
+        const childinfo = @typeInfo(typeinfo.child);
+        if (typeinfo.child == u8 and typeinfo.size != .One) {
+            return true;
+        } else if (typeinfo.size == .One and childinfo == .Array and childinfo.Array.child == u8) {
+            return true;
+        }
+        return false;
+    }
+
+    /// Pushes any string type
+    fn pushAnyString(lua: *Lua, value: anytype) !void {
+        const info = @typeInfo(@TypeOf(value)).Pointer;
+        switch (info.size) {
+            .One => {
+                const childinfo = @typeInfo(info.child).Array;
+                std.debug.assert(childinfo.child == u8);
+                std.debug.assert(childinfo.sentinel != null);
+
+                const casted: *childinfo.child = @ptrCast(@constCast(childinfo.sentinel.?));
+                if (casted.* != 0) {
+                    @compileError("Sentinel of slice must be a null terminator");
+                }
+                _ = lua.pushString(&(value.*));
```

Comment:
This `&(value.*)` is funky... is there a simpler way?

Edit: just tested, everything compiled and tested fine if I just passed `value` to pushString

Reply:
Okay I will make this change